src/kirby/components/button/button.component.scss

```diff
@@ -13,18 +13,25 @@ button {
   background-color: var(--primary);
   border-color: var(--primary);
   border-radius: var(--kirby-button-border-radius);
-  border: 1px solid;
+  border-width: 1px;
+  border-style: solid;
   color: var(--on-primary);
   cursor: pointer;
   font-size: font-size('n');
-  padding: 0.7rem 0.75rem;
+  padding: 0.7rem 1.75rem;
   transition: opacity 200ms linear;
   width: 100%;
 
   &.round {
     --kirby-button-border-radius: 2rem;
   }
 
+  &.button--secondary {
+    background-color: var(--secondary);
```

Comment:
Maybe something more specific than just `--secondary`, to avoid clashing with other components/frameworks?
+ Add suffix (`-color`, `-background-color`, `-border-color`, `border-radius` etc)
E.g. `--kirby-secondary-color`